vizro-core/examples/kpi/README.md

```diff
@@ -0,0 +1,15 @@
+# KPI dashboard
+
+This demo dashboard provides an example of a Key Performance Indicator (KPI) dashboard, designed to help users get started and extend further.
+It uses fictional budget data to demonstrate the capabilities of Vizro using real world applications.
+
+Special thanks to the [#RWFD Real World Fake Data initiative](https://opendatainitiative.io/), a community project that
+provides high-quality fake data for creating realistic dashboard examples for real-world applications.
+
+Note: The data has been additionally edited for the purpose of this example.
+
+## How to run the example locally
+
+1. Download the `Financial Consumer Complaints.csv` file from [this link](https://data.world/markbradbourne/rwfd-real-world-fake-data/workspace/file?filename=Financial+Consumer+Complaints.csv) and place it in the `data` folder
```

Comment:
Would this download the correct data or the "additionally edited for the purpose of example" data?

Also why don't we just put the data in our repo itself? Unless it's huge we can just do that.

Also don't know if you realised but the `data` folder won't appear in git if it's empty. You need a `.gitkeep` file in there. But if we can just put the data itself there then even better.

Reply:
Not necessary anymore - I've replaced the csv file with the data query that @maxschulz-COL found ⭐ 👍 https://github.com/mckinsey/vizro/pull/505/commits/636c5d9367b5daac9a0c09623a40ccffbfb18cb3

Reply:
To your question: It downloads the original data - everything that is done "additionally for the purpose of this example" lives in the `_clean_data_and_add_columns` helper function :)

Reply:
For future reference I added the iris dataset to our docs so that it can be downloaded automatically (see link in [this section](https://vizro.readthedocs.io/en/latest/pages/user-guides/data/#supply-directly)) so it's definitely possible and ok to add a .csv file to our repo in theory anyway.

Downloading the data directly in the `pd.read_csv` like you do now is even better though. So long as it's ok by the website we're hitting and we don't get hit by some sort of rate limiting e.g. when it runs repeatedly in our tests.